pages/1.10/administering-clusters/add-a-node/index.md

```diff
@@ -13,21 +13,21 @@ enterprise: false
 
 You can add agent nodes to an existing DC/OS cluster. 
 
-Agent nodes are designated as [public](/1.10/overview/concepts/#public-agent-node) or [private](/1.10/overview/concepts/#private-agent-node) during installation. By default, they are designated as private during [GUI][1] or [CLI][2] installation.
+Agent nodes are designated as [public](/1.10/overview/concepts/#public-agent-node) or [private](/1.10/overview/concepts/#private-agent-node) during installation. By default, they are designated as private during [installation][1].
 
 **Note:**
 - DC/OS supports adding/replacing agent nodes but does not support adding master nodes.
 
 ### Prerequisites:
 
-*   DC/OS is installed using the [custom](/1.10/installing/oss/custom/) installation method.
-*   The archived DC/OS installer file (`dcos-install.tar`) from your [installation](/1.10/installing/oss/custom/gui/#backup).
-*   Available agent nodes that satisfy the [system requirements](/1.10/installing/oss/custom/system-requirements/).
+*   DC/OS is installed using the [custom](/1.10/installing/) installation method.
+*   The archived DC/OS installer file (`dcos-install.tar`) from your [installation](/1.10/installing/gui/#backup).
```

Comment:
still grabbing partial links

Reply:
/1.10/installing/gui/#backup is a broken link. Replace it with /1.10/installing/
GUI installer is deprecated so we won't see it in our docs.